linera-service/src/node_service.rs

```diff
@@ -1018,17 +925,17 @@ where
             })?;
             util::wait_for_next_round(&mut stream, timeout).await;
         };
-        Ok(async_graphql::Response::new(hash.to_value()))
+        let response = async_graphql::Response::new(hash.to_value());
+        Ok(serde_json::to_vec(&response)?)
```

Comment:
I'm not sure if this is really an improvement - sure the type is `Vec<u8>` but it is a serialized JSON. If it can't be anything else I think the previous type was better b/c it more precisely defined how to handle the data. Right now callers have to read the docs, previously it was in the type.

Reply:
Actually, wouldn't it be more consistent to not return the block hash here, and instead return the `response` from the `QueryOutcome`, just like in line 903 above?

Reply:
please guys read the summary